### DISCUSSION TITLE
How can i run Visualizer locally?

### DISCUSSION THREAD

**Discussion by someshwar1** [statelyai/xstate#3614] [Q&A]
How can i run Visualizer locally?  or i want check my state machine locally.  please provide sample code if available

**Comment by bmadone** [comment]
I use VS Code extension for it, it is solve your problem?
<img width="654" alt="CleanShot 2022-10-24 at 12 44 37@2x" src="https://user-images.githubusercontent.com/23120972/197498275-cc6f7345-e468-4b7a-bc78-1c102c508d47.png">
